docs/experiment/apis/management-api.md

```diff
@@ -55,6 +88,8 @@ Endpoints that list resources such as `/experiments/list` will only return a lim
 
 ------
 
+## Experiment Endpoints
```

Comment:
```suggestion
## Experiment endpoints
```

I know it takes getting used to -  we use sentence case in headings

Reply:
updated!